src/compas_viewer/scene/collectionobject.py

```diff
@@ -27,10 +27,14 @@ def __data__(self):
 class CollectionObject(ViewerSceneObject, GeometryObject):
     """Viewer scene object for displaying a collection of COMPAS geometries."""
 
-    def __init__(self, collection: Collection, **kwargs):
-        self.collection = collection
-        super().__init__(geometry=self.collection, **kwargs)
-        self.objects = [ViewerSceneObject(item, **kwargs) for item in self.collection.items]
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        kwargs.pop("item")
```

Comment:
what does this do?

Reply:
This was to remove item from kwargs so in next line `item` is not given twice which will trigger error.

Reply:
just put item as a keyword argument in the first init